pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -2596,7 +2596,7 @@ public boolean instanceExists(String instanceName) {
     }
 
     if (tableNamesWithType.isEmpty()) {
-      throw new TableNotFoundException(tableNamesWithType + " not found.");
+      throw new TableNotFoundException("Table '" + tableName + " not found.");
```

Comment:
You might also want to include the table type if it is not null?
Also, the second quote is missing

Reply:
Fixed.